app/Links.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Links extends Model
+{
+    /**
+     * Table associated to state model
+     */
+    protected $table = 'links';
+
+    /**
+     * Primary key associated with table
+     */
+    protected $primaryKey = 'link_id';
+
+    /**
+     * Do not have eloquent create created_at and updated_at columns
+     */
+    public $timestamps = false;
+
+    /**
+     * The attributes that are mass assignable.
+     */
+    protected $fillable = [
+        'linkText'
```

Comment:
Maybe want to have two fields? 
One for the link (`https://github.com/.....etc......`) and one for the link name (`GitHub`) ?